sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -2199,9 +2247,10 @@ class DaemonXcvrd(daemon_base.DaemonBase):
     def __init__(self, log_identifier, skip_cmis_mgr=False):
         super(DaemonXcvrd, self).__init__(log_identifier)
         self.stop_event = threading.Event()
-        self.sfp_error_event = multiprocessing.Event()
+        self.sfp_error_event = threading.Event()
```

Comment:
It seems we are using it at the below:
https://github.com/sonic-net/sonic-platform-daemons/blob/master/sonic-xcvrd/xcvrd/xcvrd.py#L2391-L2392
        if self.sfp_error_event.is_set():
            sys.exit(SFP_SYSTEM_ERROR)

Reply:
But that code will never get hit, when the event get set due to error.

Reply:
Agreed, since we are killing the parent process (xcvrd) if next_state is STATE_EXIT, the above code will never get hit. This seems like a day 1 behavior though.